test/HttpApiHandlerTests.cs

```diff
@@ -73,6 +73,78 @@ private void CreateCheckStatusResponse_Throws_Exception_When_NotificationUrl_Mis
                 status["terminatePostUri"]);
         }
 
+        [Fact]
+        public void CreateCheckStatus_Returns_Corrent_CheckStatus_based_on_default_values()
+        {
+            var httpApiHandler = new HttpApiHandler(new DurableTaskExtension() { NotificationUrl = new Uri(TestConstants.NotificationUrl) }, null);
+            CheckStatus checkStatus = httpApiHandler.CreateCheckStatus(TestConstants.InstanceId, null, null);
+            Assert.NotNull(checkStatus);
+            Assert.Equal(checkStatus.Id, TestConstants.InstanceId);
+            Assert.Equal(
+                "http://localhost:7071/admin/extensions/DurableTaskExtension/instances/7b59154ae666471993659902ed0ba742?taskHub=DurableFunctionsHub&connection=Storage&code=mykey",

```

Comment:
Is the `http://localhost:7071/admin/extensions/DurableTaskExtension` based on `TestConstants.NotificationUrl`? If so, this might be a good place to reduce string duplication. I mention this mainly because Functions v2 will have a different URL prefix, and it might be nice to be able to update the constant value in one place and have these tests continue to work.

Reply:
Thank you! I will change it.